src/main/java/dinkplugin/util/ItemUtils.java

```diff
@@ -64,9 +59,17 @@ public boolean isItemNeverKeptOnDeath(int itemId) {
         return NEVER_KEPT_ITEMS.contains(itemId);
     }
 
-    public long getPrice(ItemManager itemManager, int itemId) {
-        int price = itemManager.getItemPrice(itemId);
-        return price > 0 ? price : itemManager.getItemComposition(itemId).getPrice();
+    public long getPrice(@NotNull ItemManager itemManager, int itemId) {
+        return getPrice(itemManager, itemId, null);
+    }
+
+    private int getPrice(@NotNull ItemManager manager, int itemId, @Nullable ItemComposition item) {
```

Comment:
Let's be consistent with the naming of the `ItemManager` variable as `itemManager` here

Reply:
Clarified in https://github.com/pajlads/DinkPlugin/pull/131/commits/85c73193d5bc0717e140b3a0092b6a4333db59a4